sentences/it/climate_HassClimateGetTemperature.yaml

```
@@ -3,9 +3,5 @@ intents:
   HassClimateGetTemperature:
     data:
       - sentences:
-          - "<what_is> <temp>"
-          - "[(<what_is> | che)] <temp> [c'è]"
-          - "<what_is> <temp> [(<in> | <of>)] <area>"
-          - "quanto (caldo | freddo) [(è | c'è | fa)] [(<in>|<the>) <area>]"
-          - "quanto [(è | c'è | fa)] (caldo | freddo) [(<in>|<the>) <area>]"
-          - "[(<what_is> | che)] <temp> [c'è] [<in>] <area>"
+          - "[<what_is>] <temp> [c'è] [(<in> | <of> | <the>)] [<area>]"
+          - "[(è | c'è | fa)] (caldo | freddo) [(<in> | <the>)] [<area>]"
```

Per comment by @auanasgheps 
```suggestion
          - "[quanto] [(è | c'è | fa)] (caldo | freddo) [(<in> | <the>)] [<area>]"
```